### PR TITLE
Remove closed associated adjustments button

### LINKED ISSUE #9027: Remove radio button asking for closed associated adjustments
## Description
**- As a:** enterprise user
**- On page:** `/admin/orders/XXXXX/edit` (order edit page)
**- I want to be able to do:** 
When changing the shipping method, I don't want to see the radio buttons on associated adjustments.

According to this thread: https://github.com/openfoodfoundation/openfoodnetwork/pull/7520#issuecomment-840718616 it's a leftover from Spree we can now remove

## Acceptance Criteria & Tests

1. Log in as enterprise user
2. Go to an edit order page
3. Edit the shipping method
4. See the radio buttons do not appear anymore

### PR DESCRIPTION
**Pull request by arthurtui**
Removes the 'Associated Adjustment Closed' options from the order edit
page (/admin/orders/XXXXX/edit).

#### What? Why?

- Closes #9027
- This button is a leftover from Spree that should be removed.

#### What should we test?

- Log in as enterprise user
- Go to an edit order page
- Edit the shipping method
- See the radio buttons do not appear anymore
- Update the shipping and see that fees are updated, too.

#### Release notes

Changelog Category: User facing changes

The title of the pull request will be included in the release notes.

### CODE REVIEW

**Review by mkllnk — COMMENTED** [review]
Nice, thank you!

This is good already but we can remove even more code.

https://github.com/openfoodfoundation/openfoodnetwork/blob/a3d996d5f5944206648a0bd51b14ca66eef5f156/app/assets/javascripts/admin/spree/orders/shipments.js.erb#L28-L34

https://github.com/openfoodfoundation/openfoodnetwork/blob/a3d996d5f5944206648a0bd51b14ca66eef5f156/app/controllers/api/v0/shipments_controller.rb#L31-L43

I'm just unsure about removing `@shipment.fee_adjustment.fire_events(:open)` altogether or just removing the condition and always unlocking and relocking the shipment.

@filipefurtad0 What do you think? Is there any case when you need to select Yes here? Or should we always treat as if you selected Yes?

**Comment by arthurtui** [comment]
I've made the changes assuming the shipment should always be treated as being unlocked and relocked as it should be easier to add or alter a related behaviour if needed at a later point. Is this the best way to go about it @filipefurtad0 ?

**Comment by filipefurtad0** [comment]
Hey @arthurtui, welcome :wave: 

No, @mkllnk - I think there is no use case for the "No" option, as changes only take place if "Yes" is selected.

As I understand it, this corresponds to the assumption that shipment should always be treated as being unlocked. So I think we're good here @arthurtui :+1:

File: app/controllers/api/v0/shipments_controller.rb [hunk 1]
```diff
@@ -28,19 +28,14 @@ def update
...
+        @shipment.fee_adjustment.fire_events(:open)
 
         if @shipment.update(shipment_params)
           @order.updater.update_totals_and_states
         end
 
-        if unlock == 'yes'
-          @shipment.fee_adjustment.close
-        end
+        @shipment.fee_adjustment.close
```
Comment (mkllnk) [inline app/controllers/api/v0/shipments_controller.rb:38]: The only concern I have about this is that it closes adjustments even if there were open before this action. But the same would happen if you selected yes. So I guess that we can leave it as is.

**Review by mkllnk — APPROVED** [review]
Thanks a lot for this contribution and for updating the specs as well. This is ready for testing.

**Comment by drummer83** [comment]
Hi @arthurtui, 
Thank you so much for this pull request! I would like to apologize that testing took so long. Thank you for your patience!

I have tested your changes on our staging server. Here is what I found!

**Before staging the PR**
- The changes of shipping fees only take place when 'yes' is selected:
Peek 2022-06-09 21-32

**After staging the PR**
- The radio buttons are not shown and updating the shipping method always updates the fees accordingly. 
- I have tested for enterprise user and super admin.
- Changes are also shown correctly on the order page in in the customer account.
Peek 2022-06-10 10-38

Thanks again! This is ready to go! :rocket:

**Pull request merged**